spec/syntax.md

```diff
@@ -216,17 +214,20 @@ respectively.
 
 Whitespace in _text_, including tabs, spaces, and newlines is significant and MUST
 be preserved during formatting.
-Embedding a _pattern_ in curly brackets ensures that _messages_ can be embedded into
+
+When a _pattern_ is quoted by embedding the _pattern_ in curly brackets, the
+resulting _message_ can be embedded into
 various formats regardless of the container's whitespace trimming rules.
+Otherwise, care must be taken to ensure that pattern-significant whitespace is preserved.
 
 > **Example**
 > In a Java `.properties` file, the values `hello` and `hello2` both contain
 > an identical _message_ which consists of a single _pattern_.
 > This _pattern_ consists of _text_ with exactly three spaces before and after the word "Hello":
 >
 > ```properties
-> hello = {   Hello   }
-> hello2={   Hello   }
+> hello = {{{{   Hello   }}}}
```

Comment:
To get the spaces in a properties file, the spaces need to be quoted.

It is possible to do this in the `.properties` syntax:

```
hello = \   Hello  \ 
```

My change illustrates using one method to quote the spaces using MF2's syntax. A different example would be:

```
hello  = {||}   Hello   {||}
hello2 = {|   |}Hello{|   |}
```

Reply:
I had to stare at [examples of .properties entries](https://en.wikipedia.org/wiki/.properties) to make sense of what you did. Makes sense. Wow. Who would have thought a popular file format without a spec would be so tricky?

Reply:
If we want to avoid any confusion for people not familiar with properties we can use json: `{ "msg" : "    Hello    " }`
(and they will be confused by all the extra "curlies" :-)

Or make it code (`String msg = "    Hello    "`), readable even if you are not a Java programmer.

But just kidding / nitpicking.
I think it is correct as changed.